docs/pages/management/operations/enroll-agent-into-automatic-updates.mdx

```diff
@@ -249,23 +251,25 @@ kubectl annotate statefulset/<your-agent-release> 'debug.teleport.dev/trigger-ev
 </TabItem>
 </Tabs>
 
-## Next steps
+### Suspending automatic updates
 
-You can temporarily suspend automatic updates for an agent:
+You can suspend automatic updates for an agent:
 
 <Tabs>
 <TabItem label="systemd">
 
 Disable the systemd timer:
 
 ```code
-$ systemctl disable --now teleport-upgrade.timer
+$ sudo systemctl disable --now teleport-upgrade.timer
 ```
 
+Use the `systemctl enable` to reenable automatic updates.
```

Comment:
I think we should use another copy/pastable code block here to make this slightly easier to follow.

Also, there shouldn't be a "the" in "Use the `systemctl enable`..." unless you say something like "the `systemctl enable` command."

Reply:
Yes that's the format, systemctl enable. I don't know there's another disable, enable flow to reuse

Reply:
Made a code block @ptgott , see update